changelog/2036.txt

```diff
@@ -0,0 +1,12 @@
+```release-note:change
+The TLS auth plugin no longer accepts CRLv1
```

Comment:
```suggestion
auth/cert: No longer accept CRLv1.
```

Reply:
This is the change that I think will be most breaking. I'm fine removing it, but we likely have to go through a deprecation window for this one.